### PR TITLE
When using the google pixel, the decoded bitmap is null, so need to get the drawable and convert that to a bitmap

### PR DESCRIPTION
**Pull request by premnirmal**
fixes issue https://github.com/nicholasrout/lens-launcher/issues/17

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -2,15 +2,15 @@ apply plugin: 'com.android.application'
 apply plugin: 'android-apt'
 
 android {
-    compileSdkVersion 24
-    buildToolsVersion "24.0.0"
+    compileSdkVersion 25
+    buildToolsVersion "24.0.2"
```
Comment (premnirmal) [inline app/build.gradle]: updated

**Comment by premnirmal** [comment]
Thanks @nicholasrout . Question: when do you plan on releasing?

**Comment by nicholasrout** [comment]
@premnirmal working on a few other things (improved launch animation and search) for release. Can do a quick release with this fix if it is affecting your usage?

**Comment by premnirmal** [comment]
No worries, it's not urgent. Thanks

**Comment by premnirmal** [comment]
Actually, is it possible to get a quick update with this fix in the play store? Not being able to launch the calendar app is becoming a pain

**Comment by nicholasrout** [comment]
Hi @premnirmal - I'm currently working on quite a few new features and should be done soon

**Pull request merged**